libs/pando/engine/src/optimization/simplify.ts

```
@@ -0,0 +1,108 @@
+        break
+      case 'lookup':
+      case 'thres':
+      case 'match':
+        if (n.br.every((br) => br.op === 'const')) {
+          const index = branching[op](
+            n.br.map((br) => br.ex),
+            n.ex
+          )
+          return n.x[index]
+        }
+        if (n.x.every((x) => x.op === 'const' && x.ex === n.x[0]!.ex))
```

Biome (1.9.4)

[error] Forbidden non-null assertion.

Unsafe fix: Replace with optional chain operator ?. This operator includes runtime checks, so it is safer than the compile-only non-null assertion operator


(lint/style/noNonNullAssertion)